### ISSUE TITLE
[Settings]: In Scan mode, Screen Reader focus is going unnecessary on 'Button text' inside Add new a profile page. 

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12013]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Open Narrator using 'Win+Ctrl+ Enter' key. Turn ON Narrator scan mode using 'Caps lock + Space' key.
4. Navigate to 'New Empty profile' and 'Duplicate' button using up/down arrow keys and observe the issue.

**User Experience:**
Screen Reader users UX will not be good if Screen Reader focus is going unnecessary on 'Button text if button name info is already announced with button control. 

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

**Attachments:**
[In Scan mode, Screen Reader focus is going unnecessary on 'Button text' inside Add new a profile page..zip](https://github.com/microsoft/terminal/files/7755031/In.Scan.mode.Screen.Reader.focus.is.going.unnecessary.on.Button.text.inside.Add.new.a.profile.page.zip)

### Expected Behavior

In Scan mode, Screen Reader focus should not go unnecessary on 'Button text' inside Add new a profile page as button information is already announced with control.

### Actual Behavior

In Scan mode, Screen Reader focus is going unnecessary on 'Button text' inside Add new a profile page. 
i.e. On pressing 'first' down arrow key on 'New empty profile', Screen Reader is announcing 'Create New button button'.
On pressing 2nd down arrow key Screen Reader focus goes on 'New Empty profile' text and read it.

**Observation:** Not able verify with NVDA Screen Reader as its browse mode does not work in app.
